test/src/DistanceFieldTest.cpp

```diff
@@ -74,17 +74,71 @@ TEST_F(DistanceFieldTest, should_compute_infinite_distance_for_object_off_ray_pa
     // After sending a ray in the direction of negative infinity, the distance should still be positive.
     rayTf.push_back(Mat3x4f::rotationRad(RGL_AXIS_X, M_PI).toRGL());
 
-    prepareNodes();
-
-    ASSERT_RGL_SUCCESS(rgl_graph_node_add_child(useRaysNode, raytraceNode));
-    ASSERT_RGL_SUCCESS(rgl_graph_node_add_child(raytraceNode, yieldPointsNode));
+    prepareNodes({0.0f, 10000.0f});
+    connectNodes(false);
 
     getResults();
 
+    ASSERT_EQ(outDistancesCount, rayTf.size());
     EXPECT_EQ(outDistances.at(0), NON_HIT_VALUE);
     EXPECT_EQ(outDistances.at(1), NON_HIT_VALUE);
 }
 
+TEST_P(DistanceFieldTest, should_compute_infinite_distance_for_object_out_of_max_range)
```

Comment:
```suggestion
TEST_P(DistanceFieldTest, out_of_max_range_case)
```

Reply:
@PiotrMrozik Why? I think this name better describes what the test checks